.vscode/settings.json

```diff
@@ -7,6 +7,10 @@
   },
   "rust-analyzer.cargo.features": "all",
   "rust-analyzer.check.command": "clippy",
+  "rust-analyzer.runnables.extraEnv": {
+    "AZURE_TEST_MODE": "live",
+    "AZURE_COSMOS_CONNECTION_STRING": "emulator"
```

Comment:
Oopsies.

Reply:
`.vscode/launch.json` *should* be gitignore'd, so I work around these sorts of issues by just adding my own launch file, for which you can pass all the env vars you want.

Reply:
Ah yeah, good catch. Shall I gitignore it now and remove it from the repo?

Reply:
Oh, just realized this was in `settings.json`. For now I'll just remove it. The problem with `launch.json` (though it's just an aesthetics/usability problem) is that it doesn't seem to affect the "Run Test/Debug Test" inlay links on top of the test itself, which is what this setting does. You can absolutely create a `launch.json` config for running a test under the debugger with certain env vars though, it's just a bit of a personal annoyance.

Anyway, I'll just remove this for now.

Reply:
Right, that is a downside. But you can come close with the right selection criteria e.g.,

```sh
cargo test -p azure_data_cosmos --test test_module_name specific_test_name_pattern
```

It'd be great if VSCode had layered settings within a repo, or a way to limit user settings to a specific directory, but I'm not aware of any capability.